staging/src/k8s.io/client-go/tools/auth/exec/exec.go

```diff
@@ -40,8 +41,9 @@ var codecs = serializer.NewCodecFactory(scheme)
 
 func init() {
 	metav1.AddToGroupVersion(scheme, schema.GroupVersion{Version: "v1"})
-	utilruntime.Must(v1alpha1.AddToScheme(scheme))
-	utilruntime.Must(v1beta1.AddToScheme(scheme))
+	utilruntime.Must(clientauthenticationv1alpha1.AddToScheme(scheme))
+	utilruntime.Must(clientauthenticationv1beta1.AddToScheme(scheme))
+	utilruntime.Must(clientauthenticationv1.AddToScheme(scheme))
```

Comment:
This is pre existing but please update `k8s.io/client-go/pkg/apis/clientauthentication/install.Install` to be correct and then have all places where we create one of these client authentication schemes use it.

Reply:
https://github.com/kubernetes/kubernetes/pull/102890/commits/121379cf22d3a176ad49df167a98ca80087a2e6d